<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportDeformationOptionsWidget</class>
 <widget class="QWidget" name="ExportDeformationOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QWidget" name="widget_file_options" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="gmt_format_options" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="gmt_domain_point_format_options">
        <property name="title">
         <string>Domain Point Format Options</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <property name="spacing">
          <number>1</number>
         </property>
         <property name="leftMargin">
          <number>9</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>9</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QRadioButton" name="gmt_lon_lat_radio_button">
           <property name="text">
            <string>Longitude / Latitude (GMT default)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="gmt_lat_lon_radio_button">
           <property name="text">
            <string>Lattitude / Longitude</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="strain_inclusion_options">
     <property name="title">
      <string>Include Strain Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QCheckBox" name="include_principal_strain_stretch_check_box">
        <property name="text">
         <string>Include principal strain/stretch</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="principal_strain_stretch_options" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="principal_strain_stretch_output_options">
           <property name="title">
            <string>Strain or Stretch</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="spacing">
             <number>3</number>
            </property>
            <property name="leftMargin">
             <number>18</number>
            </property>
            <property name="topMargin">
             <number>1</number>
            </property>
            <property name="rightMargin">
             <number>1</number>
            </property>
            <property name="bottomMargin">
             <number>1</number>
            </property>
            <item>
             <widget class="QRadioButton" name="principal_output_strain_radio_button">
              <property name="text">
               <string>Strain (extension is positive; compression is negative)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="principal_output_stretch_radio_button">
              <property name="text">
               <string>Stretch (1.0 + strain; is positive; can plot as ellipse)</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="principal_strain_stretch_format_options">
           <property name="title">
            <string>Format Options</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <property name="spacing">
             <number>3</number>
            </property>
            <property name="leftMargin">
             <number>18</number>
            </property>
            <property name="topMargin">
             <number>1</number>
            </property>
            <property name="rightMargin">
             <number>1</number>
            </property>
            <property name="bottomMargin">
             <number>1</number>
            </property>
            <item>
             <widget class="QRadioButton" name="principal_angle_major_minor_radio_button">
              <property name="text">
               <string>MajorAngle (-180 to +180 degrees anti-clockwise from West; 0 is East) / MajorAxis / MinorAxis</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="principal_azimuth_major_minor_radio_button">
              <property name="text">
               <string>MajorAzimuth (0 to 360 degrees clockwise from North; 0 is North) / MajorAxis / MinorAxis</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="include_dilatation_strain_check_box">
        <property name="text">
         <string>Include dilatation strain</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="strain_rate_inclusion_options">
     <property name="title">
      <string>Include Strain Rate Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QCheckBox" name="include_dilatation_strain_rate_check_box">
        <property name="text">
         <string>Include dilatation strain rate (1/sec)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="include_second_invariant_strain_rate_check_box">
        <property name="text">
         <string>Include total strain rate (1/sec)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="include_strain_rate_style_check_box">
        <property name="text">
         <string>Include strain rate style</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="deformation_output_description_group_box">
     <property name="title">
      <string>Export Description</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QLabel" name="deformation_output_description_label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../qt-resources/qt_resources.qrc"/>
 </resources>
 <connections/>
</ui>
